content/updates/2021-11/1-TA-importexport.md

```diff
@@ -7,7 +7,7 @@ pim_announcement_audience:
 ---
 
 # Importing and Exporting Table Attributes
-::: meta-data type="New" features="Governance" available="Late-November" in="EE" link-to-doc="../articles/manage-multidimensional-data-in-a-table.html#importexport-your-table-data.html"
+::: meta-data type="New" features="Governance" available="Late-November" in="EE,GE" link-to-doc="../articles/manage-multidimensional-data-in-a-table.html#importexport-your-table-data.html"
 
 Several dedicated import and export jobs for table attributes are available to adapt to your most efficient workflows. This means more autonomy for you and table attributes that become more readable and easier to enrich. Your exported table attributes will be displayed exactly the way you see them in the product edit form.  
```

Comment:
```suggestion
Several new dedicated import and export jobs for table attributes are available to enable you to download your newly created tables, in tabular format in XLS or CSV. Check out the product table values and product model table values. Utilizing these import and export jobs makes it easier to read and enrich your table attributes since your exported table attributes will be displayed exactly the way you see them in the product edit form.  
```